edge-modules/api-proxy-module/rust-sdk/azure-iot-mqtt/src/iotedge_client.rs

```diff
@@ -0,0 +1,330 @@
+use std::future::Future;
```

Comment:
q: I thought you were going use the client that @dmolokanov has created?

Reply:
I did but not here. this is arnav SDK. I am using it here: https://github.com/Azure/iotedge/pull/3338/files#diff-3ea894b65cce2edcc29f46f8563362f5R110